ontotext-yasgui-web-component/src/components/ontotext-yasgui-web-component/ontotext-yasgui-web-component.tsx

```diff
@@ -102,10 +135,30 @@ export class OntotextYasguiWebComponent {
 
     // @ts-ignore
     if (window.Yasgui) {
-      this.yasgui = this.yasguiBuilder.build(this.el, config);
+      this.yasgui = this.yasguiBuilder.build(this.hostElement, config);
       this.yasgui.addYasqeListener('query', this.onQuery.bind(this));
       this.yasgui.addYasqeListener('queryResponse', (args: EventArguments) => this.onQueryResponse(args[0], args[1], args[2]));
+
+      // Initialize render buttons styling.
+      VisualisationUtils.changeRenderMode(this.hostElement, getRenderingMode(config));
+
+      // Initialize orientation button styling.
+      const orientation = getOrientation(config);
+      VisualisationUtils.setOrientation(this.hostElement, orientation);
+      VisualisationUtils.changeOrientation(this.hostElement, orientation);
+      if (this.haveToHiddeHeader(config)) {
+        HtmlElementsUtil.getHeader(this.hostElement).classList.add('hidden');
+      } else {
+        HtmlElementsUtil.getHeader(this.hostElement).classList.remove('hidden');
+      }
+    }
+  }
+
+  private haveToHiddeHeader(yasguiConfig: YasguiConfiguration) {
```

Comment:
haveToHiddeHeader > haveToHideHeader

Reply:
this function is removed after refactoring